building.sk

```diff
@@ -760,3 +760,40 @@ caves and cliffs update part 1:
 	[any] stone ore = any stone ore, copper ore
 	[any] deepslate ore = deepslate coal ore, deepslate iron ore, deepslate copper ore, deepslate gold ore, deepslate redstone ore, deepslate emerald ore, deepslate lapis ore, deepslate diamond ore
 	[any] ore = any stone ore, any nether ore, any deepslate ore
+
+the wild update:
+	minecraft version = 1.19 or newer
+
+	# froglights
+	ochre froglight¦s = minecraft:ochre_froglight
+	pearlescent froglight¦s = minecraft:pearlescent_froglight
+	verdant froglight¦s = minecraft:verdant_froglight
+	any froglight¦s = ochre froglight, pearlescent froglight, verdant froglight
+
+	# muds
+	mud = minecraft:mud
+	muddy mangrove roots = minecraft:muddy_mangrove_roots
+	packed mud = minecraft:packed_mud
+	mud bricks = minecraft:mud_bricks
+
+	{waterloggable} {slab} mud brick slab¦s = minecraft:mud_brick_slab
+	[any] slab¦s = any slab, mud brick slab
+
+	reinforced deepslate = minecraft:reinforced_deepslate
+
+	# Sculk
+	sculk¦s = minecraft:sculk
+	
+	sculk catalyst = minecraft:sculk_catalyst
+
+	{summonable}:
+		{default} = -
+		(summonable|active) = -[can_summon=true]
+		(non-summable|no-warden|activated) = -[can_summon=false]
+	
+	{shriekable}:
+		{default} = -
+		shrieking = -[shrieking=true]
+		(quiet|non-shrieking) = -[shrieking=false]
+	
+	{waterloggable} {summonable} {shriekable} {sculk} shrieker = -shrieker
```

Comment:
This needs fixed due to the removal of {skulk}

Reply:
Got it, fixing